test/Game-test.js

```diff
@@ -0,0 +1,22 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Game = require('../src/Game');
+const Round = require('../src/Round');
+
+describe('Game', function() {
+  let game;
+
+  beforeEach(function() {
+    game = new Game();
+  });
+
+  it('should be an instance of Game', function() {
+    expect(game).to.be.an.instanceOf(Game);
+  });
+
+  it('should know what the round is', function() {
+    game.start();
```

Comment:
I don't think you have learned what spies are yet, but this would be a good place to use one. This starts the game in the terminal, which seems to prevent the test suite from finishing